_includes/specials/policy/en/05-dos.md

```diff
@@ -0,0 +1,124 @@
+#364292 included a 1MB [“megatransaction”][megatx mempool space] with
+thousands of inputs which took an abnormally long time to validate due
+to [signature verification and quadratic sighashing][rusty megatx]. An
+attacker may also make all but the last signature valid, causing the
+node to spend minutes on this transaction, only to find that it is
+garbage. During that time, a new block may have arrived but the node
+was too busy to process it. One can imagine this type of attack being
+targeted at competing miners to gain a “head start” on the next block.
+
+As such, Bitcoin Core nodes impose a maximum standard size and a
+maximum number of signature operations (or "sigops") on each
+transaction, more restricting than the block consensus limit. While
+this means some legitimate transactions may not be accepted or
+relayed, those transactions are expected to be rare. These are
+examples of _transaction relay policy_, a set of validation rules in
+addition to consensus which nodes apply to unconfirmed transactions.
+
+Some policies apply to transactions in relation to one another. A
+previous post discussed an ancestor package limit which also assists
+the block template building algorithm. Bitcoin Core nodes enforce
+symmetrical restrictions on descendant packages. Like block template
+production, the eviction algorithm is more effective when working with
+smaller packages. These two package limits [restrict the computational
+complexity][se descendant limits] of mempool insertion and deletion
+which require updating a transaction’s ancestor and descendant sets.
+
+In the last post, we mentioned the 1sat/vB minimum relay feerate
+policy ("minrelaytxfee" in Bitcoin Core), which sets a “price” for
+network transaction relay. By default, Bitcoin Core nodes do not
+accept any transactions below this feerate. In fact, Bitcoin Core
+nodes do not verify any signatures before checking the feerate
+requirements. A non-mining node doesn’t ever receive fees – they are
+only paid to the miner who confirms the transaction – and some network
+bandwidth has already been used by the time the fee is checked.
+However, fees represent a cost to the attacker. Somebody may send an
+extremely high amount of transactions that require validation and
+relay, but will at some point run out of money to pay the fees.
+
+The Replace by Fee [policy implemented by Bitcoin Core][bitcoin core
+rbf docs] requires that the replacement transaction pay a higher
+feerate than the transaction(s) it directly conflicts with, but also
+requires that it pay a higher total fee. The additional fees (the fees
+paid by the replacement subtracted by the total fees paid by all of
+the transactions it replaces) divided by the replacement transaction's
+virtual size must be at least 1sat/vB. This fee policy is not
+primarily concerned with the incentive compatibility. Rather, it
+prevents repeated replacements of the same transaction in which each
```

Comment:
Maybe something along the lines of “Rather, this incurs a minimum cost for repeated transaction replacements to curb bandwidth wasting attacks”?